src/Graphics/EasyImage.purs

```diff
@@ -0,0 +1,10 @@
+module Graphics.EasyImage where
+
+import Prelude (Unit())
+import Control.Monad.Aff (Aff())
+
+foreign import data EASY_IMAGE :: !
+
+foreign import cropInPlace :: forall e.
+                              Int -> Int -> Int -> Int ->
```

Comment:
One possible suggestion - would it be clearer to provide these x/y/w/h arguments using a record rather than as separate arguments? That would avoid any ambiguity in the order of arguments... but then perhaps it's more convenient to use the function when it's specified like this.

Just a thought.